File: Source/JavaScript/model/interactions/CloseDialogAction.ts
// Copyright (c) Cratis. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

import { InteractionAction } from './InteractionAction';

/**
 * Dismisses the innermost dialog.
 */
export interface CloseDialogAction extends InteractionAction {}

export const CloseDialogActionPropertyNames: (keyof CloseDialogAction)[] = ['kind'];

File: Source/DotNET/Model/Interactions/CloseDialogAction.cs
// Copyright (c) Cratis. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

namespace Cratis.Scene.Model.Interactions;

/// <summary>
/// Dismisses the innermost dialog.
/// </summary>
public record CloseDialogAction : InteractionAction
{
    /// <inheritdoc/>
    public override InteractionActionKind Kind => InteractionActionKind.CloseDialog;
}
